backend/api/AuthApi/index.js

```diff
@@ -34,12 +36,17 @@ const createOauthCallbackRoute = async (oauthProviderName, code, response) => {
   const oauthProvider = createOauthProvider(oauthProviderName);
   const accessToken = await oauthProvider.fetchAccessToken(oauthProvider.oauthId, oauthProvider.oauthSecret, code);
   const oauthProfile = await oauthProvider.fetchProfile(accessToken);
+  const userEmail = await oauthProvider.fetchEmail(accessToken);
 
+  oauthProfile.email = userEmail[0]['email'];
```

Comment:
`oauthProvider.fetchEmail(accessToken);` should be inside of the `githubFetchAuthorizedAccount` (because google e.g. doesn't need to fetch email separately).